### PR TITLE
Reduce whisper decoder file size with onnx export

### PR DESCRIPTION
**Pull request by csukuangfj**
Before this pull-request:
```
-rw-r--r--  1 fangjun  staff   105M Aug  7 16:22 tiny.en-decoder.int8.onnx
-rw-r--r--  1 fangjun  staff   185M Sep 20 19:13 tiny.en-decoder.onnx
```

With this pull-request:
```
-rw-r--r--  1 fangjun  staff    86M Sep 20 19:09 tiny.en-decoder.int8.onnx
-rw-r--r--  1 fangjun  staff   109M Sep 20 19:09 tiny.en-decoder.onnx
```

---

It turns out onnx saves a transposed version of `self.textDecoder.token_embedding.weight` at the output when computing `logits`. This PR removes the transposed version to reduce the file size.

---

https://github.com/openai/whisper/#available-models-and-languages
says tiny.en has 39 M parameters, whose file size is about 39 * 1e6 * 4 /1024/1024 = 148.77 MB.

Our exported ONNX model file sizes for float32 are
```
-rw-r--r--  1 fangjun  staff   109M Sep 20 19:31 tiny.en-decoder.onnx
-rw-r--r--  1 fangjun  staff    36M Aug  7 16:22 tiny.en-encoder.onnx
```

109 + 36 = 145 MB, which matches the expected file size 148.77 MB.

### CODE REVIEW

**Comment by csukuangfj** [comment]
Please re-download the whisper onnx model or re-export it with the latest code.
